crescent/internal/handle_resp.py

```diff
@@ -200,6 +192,23 @@ def _context_from_interaction_resp(interaction: CommandInteraction) -> Context:
     )
 
 
+_VALUE_TYPE_LINK: dict[OptionType | int, str] = {
+    OptionType.ROLE: "roles",
+    OptionType.USER: "users",
+    OptionType.CHANNEL: "channels",
+    OptionType.ATTACHMENT: "attachments",
+}
+
+
+def _get_option_resolved_key(option_type: OptionType | int, in_guild: bool) -> str | None:
+    option_str = _VALUE_TYPE_LINK.get(option_type)
+
+    if in_guild and option_str == "users":
+        return "members"
+
+    return option_str
```

Comment:
Just because it's in a guild doesn't guarantee that the type will be a member, right? Like if the user left the server?

Reply:
According to the discord docs "User object for the invoking user, if invoked in a DM". I think its safe to assume if this condition isn't meant it will always be a member object.
<https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object>

Reply:
Well since it's a command I guess it doesn't make sense for them to invoke a command while not in the guild lol.

Reply:
You cant invoke a slash command on a user that left but you can for a message command apparently. Gonna have to think about the best solution for this.

Reply:
I think im gonna leave user how it was before this PR. I think the most correct way to provide safety is only allowing `hikari.User` as the type hint.